index.js

```diff
@@ -22,11 +22,10 @@ module.exports = app => {
 
     if (await shouldPruneRedundantFeatures(context, config, travisBuildId)) {
       await pruneRedundantFeatures(context, repoDir.name, config, travisBuild);
-    } else if (await shouldAcceptPassingFeature(context, config, travisBuild)) {
-      await context.github.pullRequests.merge(
-        context.repo({ number: travisBuild.pull_request_number })
-      );
-
+    } else if (await shouldMergeAcceptedFeature(context, config, travisBuild)) {
```

Comment:
note that if you declare all the promises as constants before this if-else block
(e.g. `const shouldPruneRedundantFeatures = shouldPruneRedundantFeatures(context, config, travisBuildId)`)
and then let them `await` only in the if block, it allows all three to run asynchronously with each other.

The current way this is implemented, each promise will initialize and wait for its resolution before the next can start. If you find bottlenecks in performance, it may be here.